init-tools.sh

```diff
@@ -25,6 +25,12 @@ OSName=$(uname -s)
             OS=OSX
             __DOTNET_PKG=dotnet-dev-osx-x64
             ulimit -n 2048
+	    # Format x.y.z as single integer with three digits for each part
+	    VERSION=`sw_vers -productVersion| sed -e 's/\./ /g' | xargs printf "%03d%03d%03d"`
+	    if [ "$VERSION" -lt 010012000 ]; then
+	        echo error: OSX version `sw_vers -productVersion` is too old!
```

Comment:
The alignment was caused by use of tab. I fixed my profile and I use only spaces now. 
Updated message looks like this now:
error: OSX version 10.11.6 is too old. 10.12 is needed as minimum.